src/System.Text.Json/src/System/Text/Json/Writer/JsonWriterHelper.cs

```diff
@@ -172,21 +172,38 @@ internal static void ValidateNumber(ReadOnlySpan<byte> utf8FormattedNumber)
                 return;
             }
 
+            //The non digit character inside the number
```

Comment:
Nit, space after //

Reply:
@vicrdguez Our comment style is to have a space after the `//` and before the words.  If you feel like this comment adds value, please insert the space.